resources/credits/dependencies.txt

```diff
@@ -1,3 +1,3 @@
 # direct external dependencies for project :server:modules:UserReg-WS
 apns-1.0.0.Beta6.jar
-slf4j-api-1.7.16.jar
+slf4j-api-1.7.30.jar
```

Comment:
Should we remove dependencies.txt from GitHub? We don't usually commit this file.